app/models/license.rb

```diff
@@ -169,6 +176,19 @@ class License < ApplicationRecord
     errors.add :group, :license_limit_exceeded, message: "license count has exceeded maximum allowed by current group (#{group.max_licenses})"
   end
 
+  # Assert owner is not a license user (i.e. licensee)
+  validate on: %i[create update] do
+    next unless
+      user_id_changed?
```

Comment:
Create an alias for this: `owner_id_changed?`. Will make renaming the column in easier later on.